modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
> It looks likeagent-desktop, supernova, help, and edge-broker don't always do a circle:tag before deploy-kubernetes

`circle:tag` is implicit. The `circle:deploy-kubernetes` always calls `circle:tag` (assuming there's a `Dockerfile` => a docker build happened).

![image](https://cloud.githubusercontent.com/assets/52489/16814231/c5767a7a-48e9-11e6-8e3d-4300c820875f.png)